packages/browser/package.json

```diff
@@ -14,10 +14,8 @@
     "precommit": "lint-staged",
     "build": "rm -rf lib/ && tsc -p tsconfig.build.json",
     "lint": "eslint --ext .ts src",
-    "package": "pnpm build && webpack",
     "test": "jest",
     "test:coverage": "jest --silent --coverage",
-    "report": "WITH_REPORT=true pnpm package",
```

Comment:
Is this relevant? We can still keep `report` by calling `pnpm build` instead

Reply:
report is a webpack thing

Reply:
`WITH_REPORT` is just for webpack

Reply:
Oops, I didn't know that. OK then